src/mappings/comptroller.ts

```diff
@@ -1,20 +1,28 @@
 /* eslint-disable prefer-const */ // to satisfy AS compiler
 
+import { log } from '@graphprotocol/graph-ts'
 import {
   MarketEntered,
   MarketExited,
   NewCloseFactor,
   NewCollateralFactor,
   NewLiquidationIncentive,
-  NewMaxAssets,
+  // NewMaxAssets,
   NewPriceOracle,
-} from '../types/comptroller/Comptroller'
+} from '../types/Comptroller/Comptroller'
 
 import { Market, Comptroller } from '../types/schema'
-import { mantissaFactorBD, updateCommonCTokenStats } from './helpers'
+import { MANTISSA_FACTOR_BD } from './consts'
+import { updateCommonCTokenStats } from './helpers'
+import { createMarket } from './markets'
 
 export function handleMarketEntered(event: MarketEntered): void {
-  let market = Market.load(event.params.cToken.toHexString())
+  log.info('COMPTROLLER::handleMarketEntered', [])
```

Comment:
please remove the redundant logging statements. If they are for debugging, don't include them in the PR

Reply:
removed the logging statements